Add a paper section.

softwaretexnik opened the pull request:
- [x] I've read [CONTRIBUTING.md](https://github.com/mbasso/awesome-wasm/blob/master/CONTRIBUTING.md).
- [x] Description explains the issue / use-case resolved, and auto-closes the related issue(s) (https://help.github.com/articles/closing-issues-via-commit-messages/).

mbasso commented:
After some experiments I find out that travis fails the check because it take an invalid url. Basically it truncates your url at this point => `[Draft].pdf`
I think that we can avoid this problem replacing the square brackets:
- `[` => `%5B`
- `]` => `%5D`

so, the correct link will be the following:

`https://people.mpi-sws.org/~rossberg/papers/Haas,%20Rossberg,%20Schuff,%20Titzer,%20Gohman,%20Wagner,%20Zakai,%20Bastien,%20Holman%20-%20Bringing%20the%20Web%20up%20to%20Speed%20with%20WebAssembly%20%5BDraft%5D.pdf`

can you please update your PR with this link?

softwaretexnik commented:
Thanks. I've changed the URL, but it seems that travis stills fails due to the URL. I found another reference for the PDF and changed the URL accordingly which let the travis build succeed.

mbasso commented:
Awesome, thank you!
